cabot/cabotapp/models.py

```diff
@@ -934,6 +940,39 @@ def _run(self):
         return result
 
 
+class TCPStatusCheck(StatusCheck):
+
+    @property
+    def check_category(self):
+        return "TCP Check"
+
+    @property
+    def description(self):
+        return 'Monitoring {0}:{1}'.format(self.address, self.port)
```

Comment:
But what if we cannot connect? I feel like the description would appear misleading